### PR TITLE
Image,input

### PR DESCRIPTION
**Pull request by AswathyVNair**
IMAGE:
1. Responsive image
responsive

2. Round image
round

INPUT:
1. Input box
input-box

2.Input box with validation
input-box with validation

3. Input box with icon
input-box-icon

4. Checkbox
checkbox

5. Radio
radio

6.Slider
slider

7.Text box
textbox

### CODE REVIEW

**Review by itsmebipul00 — APPROVED** [review]
Nice work done with checkbox

File: input/input.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Compopnent Library</title>
```
Comment (Megha-Joshi) [inline input/input.html:7]: Correct the spelling of Component

**Review by Megha-Joshi — APPROVED** [review]
Good to merge

**Review by Shubhamkr2610 — APPROVED** [review]
Code is clean. Everything is good to me. You can merge

**Review by Shubhamkr2610 — APPROVED** [review]
good to go . everything is fine to me

File: input/input.html (hunk 1 above)
Comment (AswathyVNair) [reply input/input.html:7]: Thanks for pointing that out.Will correct.

**Pull request merged**